<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FirstDialog</class>
 <widget class="QDialog" name="FirstDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>321</width>
     <height>61</height>
    </rect>
   </property>
   <property name="plainText">
    <string>Welcome to Carbon Tracker, the app you get to know you carbon emmisions.
First we ask you some questions:</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>100</y>
     <width>198</width>
     <height>150</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Question 1 (when multiple are possible)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="Answer1">
        <property name="text">
         <string>Answer 1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="Answer2">
        <property name="text">
         <string>Answer 2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="Answer3">
        <property name="text">
         <string>Answer 3</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="Answer4">
        <property name="text">
         <string>Answer 4</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Check">
        <property name="text">
         <string>Check</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
